packages/react-components/react-carousel-preview/stories/src/Carousel/CarouselActionCards.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  Button,
+  makeStyles,
+  tokens,
+  Image,
+  Persona,
+  Dropdown,
+  Option,
+  useId,
+  Switch,
+} from '@fluentui/react-components';
+
+import { MoreHorizontalRegular, DocumentLinkRegular } from '@fluentui/react-icons';
+import {
+  Carousel,
+  CarouselCard,
+  CarouselNav,
+  CarouselNavButton,
+  CarouselNavContainer,
+  CarouselProps,
+  CarouselSlider,
+} from '@fluentui/react-carousel-preview';
+import * as React from 'react';
+
+const useClasses = makeStyles({
+  actionCard: {
+    borderRadius: tokens.borderRadiusLarge,
+    overflow: 'hidden',
+    textAlign: 'center',
+    maxWidth: '350px',
+    margin: '0px 6px',
+    boxShadow: tokens.shadow16,
+  },
+  imageContainer: {
+    position: 'relative',
+  },
+  image: {
+    maxHeight: '200px',
+  },
+  imageButton: {
+    position: 'absolute',
+    left: '12px',
+    bottom: '12px',
+  },
+  info: {
+    display: 'flex',
+    flexDirection: 'row',
+    padding: '12px',
+    width: 'auto',
+    alignContent: 'center',
+    justifyContent: 'space-between',
+  },
+  moreButton: {},
+});
+
+const swapImage = 'https://fabricweb.azureedge.net/fabric-website/assets/images/wireframe/image-square.png';
+
+const ActionCard: React.FC<{ secondaryText: string; text: string }> = props => {
+  const { secondaryText, text } = props;
+  const classes = useClasses();
+
+  return (
+    <CarouselCard focusMode="tab-exit" className={classes.actionCard}>
+      <div className={classes.imageContainer}>
+        <Image fit="cover" className={classes.image} src={swapImage} role="presentation" />
+        <Button className={classes.imageButton} icon={<DocumentLinkRegular />} aria-label={'Go to document'} />
+      </div>
+      <div className={classes.info}>
+        <Persona textPosition="after" name={text} presence={{ status: 'available' }} secondaryText={secondaryText} />
+        <Button className={classes.moreButton} icon={<MoreHorizontalRegular />} aria-label={'More options'} />
+      </div>
+    </CarouselCard>
+  );
+};
+
+const alignmentOptions = ['center', 'end', 'start'];
+
+export const ActionCards = () => {
+  const dropdownId = useId('dropdown');
+  const [alignment, setAlignment] = React.useState<CarouselProps['align']>();
+  const [whitespace, setWhitespace] = React.useState<boolean>(false);
+  return (
+    <div>
+      <label id={dropdownId}>Alignment:</label>
+      <Dropdown
+        aria-labelledby={dropdownId}
+        placeholder="Select an alignment"
+        onOptionSelect={(event, option) => {
+          setAlignment(option.optionText as CarouselProps['align']);
```

Comment:
I didn't notice any changes visually when I played with this - should I have noticed something?

Reply:
This is more noticeable when 'whitespace' is true or depending on the viewport sizing, i.e. if you can fit 1.5 cards in view, the current card would be either start/center/end, or with whitespace true it will orient differently at the start/end card.